submissions/TatyanaMolchanova/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" type="image/png" href="./images/favicon.png">
+</head>
+<body>
+    <nav class="nav">
+        <ul class="nav-list">
+            <li class="nav-item">
+                <a href="#" class="nav-link">
+                    <span>Mail</span>
+                </a>
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link">
+                    <span>Images</span>
+                </a>
+            </li>
+            <li class="nav-item" style="position:relative;">
```

Comment:
You forget about these inline-styles)

Reply:
Remove it and check the absense of other inline styles.